### PR TITLE
Added Popptr1 to save some bytes and ease parameter extraction

### PR DESCRIPTION
**Pull request by IrgendwerA8**
Further savings are possible by exchange of assignments to ptr-variables  like libsrc/cbm/rwcommon.s "ptr1<->ptr2"  (but here you have to take care of dependent files).

### CODE REVIEW

**Comment by IrgendwerA8** [comment]
Just noticed that there is may an issue with "strpbrk.s" (use of X-register). This leads to the question what happened to the function anyway: The description in the "funcref.sgml" is commented, there is no prototype to include and even the fastcall convention isn't applied.
Time to fix or remove?

**Comment by IrgendwerA8** [comment]
Revert change on "strpbrk.s". Together with documentation update, calling convention change and prototype introduction this is a nice extra story.

**Comment by oliverschmidt** [comment]
Nice :-))

> Together with documentation update, calling convention change and prototype introduction this is a nice extra story.

Full ACK.

File: libsrc/geos-common/drivers/fio_module.s [hunk 1]
```diff
@@ -13,7 +13,7 @@
 FILEDES         = 3             ; first free to use file descriptor
 
             .importzp ptr1, ptr2, ptr3, tmp1
-            .import addysp, popax
+            .import addysp, popax, poptr1
```
Comment (greg-king5) [inline libsrc/geos-common/drivers/fio_module.s]: typo: popptr1

**Comment by IrgendwerA8** [comment]
Sh... don't mess around with foreign targets (wonder why there wasn't a compile problem).
Do you mind if I would fix this in my "Saved 6 bytes by using popptr1" pull request?

**Comment by oliverschmidt** [comment]
> I have to hold back myself to mess around with foreign targets.

You might have a point here after all ;-)) ... see  https://github.com/cc65/cc65/commit/e949a3e64283e65b00ba8311bc9cec7fcb6ff681

**Pull request merged**